geo-types/src/geometry/coordinate.rs

```diff
@@ -25,12 +25,15 @@ use approx::{AbsDiffEq, RelativeEq, UlpsEq};
 /// [vector space]: //en.wikipedia.org/wiki/Vector_space
 #[derive(Eq, PartialEq, Clone, Copy, Debug, Hash, Default)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct Coordinate<T: CoordNum = f64> {
+pub struct Coord<T: CoordNum = f64> {
     pub x: T,
     pub y: T,
 }
 
-impl<T: CoordNum> From<(T, T)> for Coordinate<T> {
+#[deprecated(note = "Renamed to `geo_types::Coord` (or `geo::Coord`)")]
+pub type Coordinate<T> = Coord<T>;
```

Comment:
```suggestion
pub type Coordinate<T = f64> = Coord<T>;
```

Without this suggested change, when building a client application (abstreet), I hit this error:

```
error[E0107]: missing generics for type alias `geo_types::Coordinate`
  --> /Users/mkirk/.cargo/registry/src/github.com-1ecc6299db9ec823/contour-0.7.0/src/contour.rs:8:26
   |
8  | pub type Pt = geo_types::Coordinate;
   |                          ^^^^^^^^^^ expected 1 generic argument
   |
```


I (similarly to you I think), assumed the default trait implementation would translate to the type alias, but it looks not to be the case.